frontend/public/components/storage/create-pvc.tsx

```diff
@@ -81,17 +63,13 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
   };
 
   handleStorageClass = storageClass => {
-    this.setState({ storageClass }, this.onChange);
+    this.setState({ storageClass: _.get(storageClass, 'metadata.name'), storageClassAccessMode: _.get(storageClass, ['metadata', 'annotations', 'storage.alpha.openshift.io/access-mode'])}, this.onChange);
```

Comment:
Had to use get here to protect when the storage class was null (when user chooses 'no storage class' option).